src/Form/form-control.mdx

```diff
@@ -163,6 +162,160 @@ or [select attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element
 }
 ```
 
+## Input masks
+Paragon uses the [react-imask](https://www.npmjs.com/package/react-imask) library,
+which allows you to add masks of different types for inputs.
+To create your own mask, you need to pass the required mask pattern (`+{1}(000)000-00-00`) to the `inputMask` property. <br />
```

Comment:
The US-based example phone number mask should likely be `+{1} (000) 000-0000` so that phone numbers are formatted like `+1 (555) 555-5555`, which is typical for a US-based phone number.

Reply:
Corrected, thanks